app/views/slides/templates/_standard.html.erb

```diff
@@ -17,7 +17,7 @@
 <% end %>
 
 <% if slide.foreground_type == 'video' %>
-  <video id="js-foreground-video" src="<%= slide.foreground %>" class="ui-slide-foreground" loop style="width: 100%; height: 100%;"></video>
+  <video controls muted id="js-foreground-video" src="<%= slide.foreground %>" class="ui-slide-foreground" loop style="width: 100%; height: 100%;"></video>
```

Comment:
Add a code comment that notes muted attribute and identifies source url as reference.